pyproject.toml

```diff
@@ -28,9 +28,9 @@ dependencies = [
     "datasette>=0.64",
     "doc8>=1.1",
     "email-validator>=1.0.3", # pydantic[email]
+    "frictionless>=4.40,<5",
```

Comment:
Alphabetized and bumped to 4.40 to match what we have in the archiver repository.